client/app/components/category_list.vue

```diff
@@ -3,19 +3,42 @@
 
     block content
         h1 {{ 'my_accounts category title' | t }}
-        article {{ 'my_accounts coming_soon' | t }}
-
-        ul
-            li
-                button(v-on:click="openDialog") open dialog
+        div.konnectors-list
+            konnector-item(
+                v-for="konnector in konnectorsList",
+                v-if="!konnector.accounts.length",
+                :title="konnector.name",
+                :subtitle="(konnector.category + ' category') | t",
+                :backgroundCSS="konnector.color.css",
+                :enableDefaultIcon="true",
+                :iconName="konnector.slug",
+                link="#")
 </template>
 
 <script>
+    import konnectorItemComponent from './list_item'
+
     export default {
-      methods: {
-          openDialog () {
-              this.$emit('open-dialog', 'dialog-1')
-          }
-      }
+        data: function () {
+            return {
+                konnectorsList: window.initKonnectors
```

Comment:
my main concern is really about testing: we shouldn't rely on external resources, so having them wrappeed in a helper that can be easily mocked is probably a better solution. But this code will be converted, so don't worry, it was just a remark, no a real need for this PR :)